Unbox.swift

```diff
@@ -674,34 +666,78 @@ private extension Unboxer {
             throw UnboxError.InvalidData
         }
     }
+
+    static func performUnboxingWithDictionary<T: Unboxable>(dictionary: UnboxableDictionary, context: Any? = nil, allowInvalidElements: Bool = false) throws -> T {
+        return try Unboxer(dictionary: dictionary, context: context).performUnboxing(allowInvalidElements)
+    }
+
+    static func performUnboxingWithDictionaries<T: Unboxable>(dictionaries: [UnboxableDictionary], context: Any? = nil, allowInvalidElements: Bool = false) throws -> [T] {
+        return try dictionaries.map {
```

Comment:
I tried it and I don't think it will work like that. Silencing the error will exclude that item from the collection completely even if it fails on one property. If it fails on one property, we still want it in the collection. I believe `allowInvalidElements` has to get to get to `throwIfFailed` in some way, but it won't this way.